### ISSUE TITLE
Cron jobs stop working (Can an error inside the time function result in job stopping?)

### ISSUE THREAD

**Issue by franky-continu** [kelektiv/node-cron#585]
### Description

Guys for some time now, about 3 weeks we have been experiencing cron job stopping. We can't identify and there is talks to move away from this library.

I'm sure this is an easy thing, this library has been the most robust one ever.

I coded our scheduler with it and it served us for 6 years. I'm not in the issue right now, but I want to collaborate.

We have a package declaration of 1.6.0 (`"cron": "^1.6.0",`).

We user an every minute one that stops calling our notifications processing. (or so we think).

Almost all or our jobs are the same (they just change in the rest resource being called)

```javascript
function schedule() {
  cronJob = cron.job("0 */1 * * * *", async function() {
    log.info('Scheduled Job ' + name + ' called');
    const api_resource = process.env.API_BASE_URL + 'v3/notifications/process';
    const now = new Date();
    const opts = {
      headers: {'Authorization': 'TOKEN'}
    };

    try {
      await axios.post(api_resource, {}, opts);
      log.info('Scheduled Job ' + name + ' triggered successfully.');
    } catch (error) {
      log.error('Scheduled Job ' + name + ' was not posted. Error was', error);
    }
  });
  cronJob.start();
}
```

We dir list all jobs, and for each we call the function schedule.

I understand the lack of items but any pointers, any ideas would be appreciated it. Thank you so much.

The bottom line is that every so often we need to redeploy our api to restart our scheduler.

**Comment by albertosantini** [comment]
@franky-continu Try "1 */1 * * * *". 
Zero second randomly seems problematic.

@Hexagon Thakns for the suggestion. I will give a try.

**Comment by franky-continu** [comment]
> @franky-continu Try "1 */1 * * * *". Zero second randomly seems problematic.
> 
> @Hexagon Thanks for the suggestion. I will give a try.

Thank you so much for the responses.


Sorry, what is the difference between Zero and 1 in terms of randomly. I'm setting to call notifications on the 0 second of every minute. Perhaps put *. but We usually set it on the dot (as in zero 0).

I'm not sure I follow the `it being` random thing.

**Comment by intcreator** [comment]
if you're still having an issue could you write a test case that illustrates it? one of these should be fairly easy to adapt: https://github.com/kelektiv/node-cron/blob/master/tests/cron.test.js

**Comment by sheerlox** [comment]
closing because stale and cannot reproduce, but feel free to re-open if you're still encountering this issue with the latest version!
